### PR TITLE
feat(combobox): add fixed-dropdown support (VIV-1739)

### PR DESCRIPTION
**Pull request by RichardHelm**
Also removed the outline that shouldn't be there

### CODE REVIEW

File: libs/components/src/lib/combobox/combobox.scss [hunk 1]
```diff
@@ -43,15 +44,17 @@
 	@include connotation.connotation(combobox);
 	@include appearance.appearance;
```
Comment (rachelbt) [inline libs/components/src/lib/combobox/combobox.scss:45]:
```suggestion

```

File: libs/components/src/lib/combobox/combobox.scss [hunk 2]
```diff
@@ -43,15 +44,17 @@
...
 
+	max-height: var(variables.$combobox-height, 408px);
 	padding: 4px;
 	border-radius: #{border-radius-variables.$border-radius-normal};
 	background-color: var(#{appearance.get-appearance-token(fill)});
```
Comment (rachelbt) [inline libs/components/src/lib/combobox/combobox.scss:50]:
```suggestion
```
same :)

File: libs/components/src/lib/combobox/ui.test.ts [hunk 3]
```diff
@@ -1,30 +1,38 @@
...
+		template: `
+			<style>
+				#wrapper {
+					display: grid;
+					width: 300px;
+					gap: 8px;
+					padding: 8px;
+				}
+			</style>
+			<vwc-combobox></vwc-combobox>
```
Comment (rachelbt) [inline libs/components/src/lib/combobox/ui.test.ts:26]: 👍

**Review by rachelbt — COMMENTED** [review]
left 2 small comments.
LGTM

**Review by rachelbt — DISMISSED** [review]

**Comment by RichardHelm** [comment]
@rachelbt Done 👍

**Review by rachelbt — APPROVED** [review]

**Pull request merged**